auth/src/main/res/values/strings.xml

```diff
@@ -32,9 +31,8 @@
         password.</string>
     <string name="error_email_does_not_exist">That email address doesn\'t match an existing account</string>
     <string name="welcome_back">Welcome back!</string>
-    <string name="next">Next</string>
     <string name="trouble_signing_in">Trouble signing in?</string>
-    <string name="sign_in">Sign in</string>
+    <string name="sign_in_default">Sign in</string>
```

Comment:
This was a nasty one! `WelcomeBackPasswordPrompt` kept using the word "Start" everywhere which was extremely confusing. Turns out our [sample was accidentally overriding](https://github.com/firebase/FirebaseUI-Android/blob/master/app/src/main/res/values/strings.xml#L15) this string resource. Since I think this could be a common occurrence, I've changed the name to `sign_in_default`.